src/DataGrid.tsx

```diff
@@ -257,6 +257,7 @@ function DataGrid<R, SR, K extends Key>(
    * The identity of the wrapper function is stable so it won't break memoization
    */
   const selectRowWrapper = useLatestFunc(selectRow);
+  const selectAllRowsWrapper = useLatestFunc(selectAllRows);
```

Comment:
May be we should use `Latest` suffix instead of `Wrapper` 🤔

Reply:
Yeah